tests-behat/bootstrap/Context.php

```diff
@@ -426,19 +425,6 @@ public function iClickIconUsingCss($arg1)
         $icon->click();
     }
 
-    /**
-     * Wait for an element, usually an auto trigger element, to show that loading has start"
-     * Example, when entering value in JsSearch for grid. We need to auto trigger to fire before
-     * doing waiting for callback.
-     * $arg1 should represent the element selector for jQuery.
-     *
-     * @Then I wait for loading to start in :arg1
-     */
-    public function iWaitForLoadingToStartIn($arg1)
-    {
-        $this->getSession()->wait(2000, '$("' . $arg1 . '").hasClass("loading")');
```

Comment:
very badly designed check - be must wat until loading is complete, not started (which we may miss completely)